client/scss/components/_find_us.scss

```diff
@@ -0,0 +1,39 @@
+//* @define find-us;
+
+.find-us {
+  @include font('HNL5');
+  border-top: 1px solid color('charcoal');
+  padding-top: 1.6rem;
+  color: color('white');
+
+  .icon {
+    display: block;
+    float: left;
+    width: 14px;
```

Comment:
It seems we're repeating this all over the shop - do you think it's worth us defining a default size?